### ISSUE TITLE
FSR2_LockInputLuma is created with display size but only used as render size

### ISSUE THREAD

**Issue by Akatsukihh** [GPUOpen-Effects/FidelityFX-FSR2#91]
Hi, 
Thanks for this creating this great technology.

It seems FSR2_LockInputLuma is created with display size but only used as a render size texture.
is it just a typo in the code?
Did not have any bug with this problem aside wasting some VRAM.

Thanks.

**Comment by fishingGrapes** [comment]
Yeah, the [the diagram](https://github.com/GPUOpen-Effects/FidelityFX-FSR2/blob/master/docs/media/super-resolution-temporal/algorithm-structure.svg) seems to mention that it is supposed to be render size. Then again, it also shows the Lock masks and Luma history in render size, which I don't think is correct.
